<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTableResult</class>
 <widget class="QDialog" name="DialogTableResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>605</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="10,0">
   <item>
    <widget class="QLabel" name="labelTableResult">
     <property name="text">
      <string>&lt;br&gt;&lt;center&gt;Розрахунок згідно ДСТУ EN ISO 1182:2022 п.7.3.1&lt;/center&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
     <item>
      <widget class="QPushButton" name="buttonTableResultSave">
       <property name="text">
        <string> Зберегти результат </string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/Save.ico</normaloff>:/icons/Save.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonTableResultExit">
       <property name="text">
        <string> Вихід </string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/exit.ico</normaloff>:/icons/exit.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
